src/Categories/Morphism/Idempotent/Bundles.agda

```diff
@@ -0,0 +1,58 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category.Core
+
+-- Bundled versions of Idempotents, as well as maps between idempotents
+module Categories.Morphism.Idempotent.Bundles {o ℓ e} (𝒞 : Category o ℓ e) where
+
+open import Level
+
+open import Categories.Morphism.Idempotent 𝒞
+open import Categories.Morphism.Reasoning 𝒞
+
+open Category 𝒞
+open HomReasoning
+open Equiv
+
+--------------------------------------------------------------------------------
+-- Bundled Idempotents, and maps between them
+
+record BundledIdempotent : Set (o ⊔ ℓ ⊔ e) where
```

Comment:
I'm not sure about putting the word `Bundled` in the record name. I think it should just be `Idempotent` - and it is the responsibility of the user to either choose a representation, or import things qualified.